README.rst

```diff
@@ -310,6 +310,8 @@ The agent sends the following
 
 - After the registration is successfully completed: ``post-registration``
 - After the registration failed: ``registration-failed``
+- After the start of the agent after a bootup of the device: ``bootup``
+- After the 2nd and all following starts of the agent after a bootup: ``restart``
```

Comment:
What about:

- When the device completes the first start after booting up: ``bootup``
- After any subsequent restarts are completed: `restarts`

Reply:
* The device doesn't complete the start, but the agent
* I would name the event `restart`, not `restarts`, because it is related to one single restart of the agent

So what about:

* When the agent first starts after the bootup of the device: `bootup`
* After any subsequent restart: `restart`

Reply:
yes, restarts was a typo, sorry.

Reply:
What do you think about my suggestion?

Reply:
@nemesisdesign I updated my last commit with the event description I suggested.